Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -478,11 +478,11 @@
 				G_3234630030493 /* FileGroup */,
 				G_4661500274312 /* Framework */,
 				G_1952740716080 /* Frameworks */,
+				G_8268950006174 /* iMessage */,
+				G_1646573205915 /* iMessage MessagesExtension */,
 				G_8620238527590 /* Products */,
 				G_7189434949822 /* Resources */,
 				G_6651250437419 /* StandaloneFiles */,
-				G_8268950006174 /* iMessage */,
-				G_1646573205915 /* iMessage MessagesExtension */,
```

Comment:
These seem to be related to a change that made it into master?

Reply:
Yes, it seems since https://github.com/yonaskolb/XcodeGen/pull/316, tests haven't been failing properly on CI.
That means the required fixture changes weren't reported as a failure as part of https://github.com/yonaskolb/XcodeGen/pull/341.

The test framework has been fixed, and this file has been update in master now 👍